packages/itwin/breakdown-trees/src/test/TreeComponents.test.tsx

```diff
@@ -116,6 +117,17 @@ describe("TreeComponent tests.", () => {
 
   it("TreeNodeWrapper renders correctly", () => {
     const node = moq.Mock.ofType<TreeModelNode>();
+    const propertyRecord = moq.Mock.ofType<PropertyRecord>();
+    const propertyValue = moq.Mock.ofType<PropertyValue>();
+    const propertyDescription = moq.Mock.ofType<PropertyDescription>();
+    propertyValue.setup((x) => x.valueFormat).returns(() => 0);
+    propertyDescription.setup((x) => x.name).returns(() => "propertyRecName");
+    propertyDescription.setup((x) => x.displayLabel).returns(() => "propertyRecDisplayLabel");
+    propertyDescription.setup((x) => x.typename).returns(() => "propertyRecTypeName");
+    propertyRecord.setup((x) => x.value).returns(() => propertyValue.object);
+    propertyRecord.setup((x) => x.property).returns(() => propertyDescription.object);
+    node.setup((x) => x.depth).returns(() => 1);
+    node.setup((x) => x.label).returns(() => propertyRecord.object);
```

Comment:
I think simply creating a `PropertyRecord` through its constructor or `PropertyRecord.fromString` would be easier more readable.